Lib/lib2to3/pgen2/tokenize.py

```diff
@@ -74,10 +74,11 @@ def maybe(*choices): return group(*choices) + '?'
 Single3 = r"[^'\\]*(?:(?:\\.|'(?!''))[^'\\]*)*'''"
 # Tail end of """ string.
 Double3 = r'[^"\\]*(?:(?:\\.|"(?!""))[^"\\]*)*"""'
-Triple = group("[ubUB]?[rR]?'''", '[ubUB]?[rR]?"""')
+_litprefix = r"(?:[uUrRbB]|[rR][bB]|[bBuU][rR])?"
+Triple = group(_litprefix + "'''", _litprefix + '"""')
 # Single-line ' or " string.
-String = group(r"[uU]?[rR]?'[^\n'\\]*(?:\\.[^\n'\\]*)*'",
-               r'[uU]?[rR]?"[^\n"\\]*(?:\\.[^\n"\\]*)*"')
```

Comment:
"String" didn't previously include `b` but this is suspicious to me. Why wouldn't it? That way `b''` literals were only PseudoTokens (via ContStr), not actual Tokens (via String).